source/admin_manual/list_of_events.rst

```diff
@@ -1113,6 +1113,29 @@ IMAP Command
    .. Note:: This event is currently not sent for pre-login IMAP commands.
 
 
+IMAP ID Command
+^^^^^^^^^^^^^^^
```

Comment:
I don't think this needs its own title? Could be just under "IMAP Client"? There's "IMAP Command" because it has its own event that is inherited by other events, but that's not a case for IMAP ID.